install-labkey.bash

```diff
@@ -960,6 +976,30 @@ function step_tomcat_service_standard() {
       fi
     fi
 
+    # Create Labkey Service script - prevents unexpected shutdown of LabKey service during upgrades etc.
+    NewScript="$TOMCAT_INSTALL_HOME/bin/labkey_service.sh"
```

Comment:
was 'NewScript' a placeholder name?

Reply:
its just a local var used for the HEREDOC file - granted not a great name, but I'm adverse to changing this as doing so would require re-running deployment tests.